doc/release-notes/release-notes-0.10.0.md

```diff
@@ -1,6 +1,6 @@
 Bitcoin Core version 0.10.0 is now available from:
 
-  https://bitcoin.org/bin/0.10.0/
+  https://bitcoin.org/bin/bitcoin-core-0.10.0/
```

Comment:
Shouldn't this pont to `https://bitcoin.org/bin/insecure/bitcoin-core-0.10.0` instead?

Please fix the rest of them as well with a scripted diff:
```bash
-BEGIN VERIFY SCRIPT-
sed -i --regexp-extended 's#(https://bitcoin\.org/bin/)([.0-9]+)#\1insecure/bitcoin-core-\2#g' ./doc/release-notes/*.md
-END VERIFY SCRIPT-
```

Reply:
> Shouldn't this pont to `https://bitcoin.org/bin/insecure/bitcoin-core-0.10.0` instead?
> 
> Please fix the rest of them as well with a scripted diff:
> 
> ```shell
> -BEGIN VERIFY SCRIPT-
> sed -i --regexp-extended 's#(https://bitcoin\.org/bin/)([.0-9]+)#\1insecure/bitcoin-core-\2#g' ./doc/release-notes/*.md
> -END VERIFY SCRIPT-
> ```

Hi @l0rinc, thanks for your suggestion. however I think the directory you provided (https://bitcoin.org/bin/insecure/) only includes limited releases that contain known security flaws and should not be used, according to the README.
The original repository that I use (https://bitcoin.org/bin/) should include the full releases that all the other release notes have already correctly referred to. Only the link in 0.10.0 seems to be wrong.

Could you please check again?

Reply:
https://bitcoin.org/bin/insecure/bitcoin-core-0.10.0 contains everything that https://bitcoin.org/bin/bitcoin-core-0.10.0 contains - and the rest of the <0.10.0 release links are also wrong and should be fixed.

Please squash the commits (we need a clean history) and fix all of them via a single [scripted-diff](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#scripted-diffs) commit.

Approach NACK